utils/cligen/Makefile

```diff
@@ -0,0 +1,66 @@
+#
+# Copyright (C) 2020-2023 Olof Hagsand and Rubicon Communications, LLC(Netgate)
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=cligen
+PKG_VERSION:=6.4.0
+PKG_RELEASE:=1
+PKG_MAINTAINER:=Olof Hagsand <olof@hagsand.se>, Philip Prindeville <philipp@redfish-solutions.com>
+PKG_LICENSE:=Apache-2.0
+PKG_LICENSE_FILES:=LICENSE.md
+
+PKG_SOURCE_URL:=https://github.com/clicon/$(PKG_NAME)/archive/refs/tags
+PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
+PKG_SOURCE_URL_FILE:=$(PKG_VERSION).tar.gz
+PKG_HASH:=d12c43c2e1ac2c4c564b2e41cdc9902b540dd18a263af1dc4a6ed47cd08e102c
+
+# Call original “make install” with prefix set to PKG_INSTALL_DIR
+PKG_INSTALL:=1
+
+include $(INCLUDE_DIR)/package.mk
+
+# Package definition; instructs on how and where our package will appear in the overall configuration menu ('make menuconfig')
+define Package/cligen
+	SECTION:=utils
+	CATEGORY:=Utilities
+	URL:=http://www.cligen.se
+	TITLE:=CLIgen is a Command-Line Interface generator
+endef
```

Comment:
1. Two spaces as indentation.
2. No dependencies at all?
3. The website uses https, so we can use https here as well even though there is redirect.